### PR TITLE
Comfy UI Quickstart docker and workflows

### PR DESCRIPTION
**Pull request by congruency**
Pretty self explanatory. Please let me know if you'd like any changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,8 +34,9 @@ we will try to keep this repo as simple as possible so that everyone in the PixA
 
 ---
 ## Breaking News 🔥🔥!!
+- (🔥 New) Apr. 24, 2024. 💥 ComfyUI quickstart Dockerfile added!
```
Comment (lawrence-cj) [inline README.md:37]: Could you please add a separate README.md (eg. PixArt-comfyui.md) in asset/comfyui and add a link here.

You can put the content of `## 5. ComfyUI Quickstart` into that PixArt-comfyui.md

**Comment by lawrence-cj** [comment]
Thanks a lot for your PR. Let's work together to make it better.

File: README.md (hunk 1 above)
Comment (congruency) [reply README.md]: Thank you for your review!

**Comment by congruency** [comment]
Oops, re-opening in a bit w/ a clean history

**Pull request closed without merging**